data/incipias/incipias first contact.txt

```diff
@@ -269,6 +274,8 @@ mission "Incipias: Help The Stranded 1"
 				`	"I want to help one of your stranded ships."`
 			`	Again a long pause. "Your intention-goals are welcome. Please dock to the capital-station." Along the message comes a small map of the station with the docking ports highlighted.`
 	on complete
+		log `Met with the Incipias on their space station, and gained access to their ships, which are capable of entering the gas giant that the stranded Incipias ship has taken refuge in.`
+		log "Factions" "Incipias" `The Incipias are a species of aliens that live to the northeast of Hai space. They originated on a gas giant, and resemble large, floating octopi. As they have not yet developed the hyperdrive, they are confined to a single system. Learned that they are under the protection of the Quarg, and was warned not to give them any advanced technology or interfere in their development.`
```

Comment:
I think it makes sense. Look at other faction logs; there are other details in there as well. This log only comes after gaining information enough about the Incipias, not initially after a short conversation about them/you meeting them similar to other facrtions.

Reply:
I guess my comment was unclear.

Other than the octopus thing and a little bit more evidence on the warning, I don't think my suggestion changes any content. Instead, it fixes the grammatical error in the last sentence, where it starts telling you what _you've_ done rather than who the Incipias are and what _they've_ done—something that I haven't seen in any other faction logs.

That said, here's a version without the octopus change:
```suggestion
		log "Factions" "Incipias" `The Incipias are a species of aliens that live to the northeast of Hai space. They originated on a gas giant, and resemble large, floating octopi. As they have not yet developed the hyperdrive, they are confined to a single system. They are under the protection of the Quarg, who have strictly warned you not to give them any advanced technology or interfere in their development.`
```

Reply:
With roadrunner's above comment - I'd keep the "Learned" in there if we don't do first/second person; other dialogs use this specific phrasing too ("Learned...").

Reply:
This suggestion has the same problem as the one above. It would probably be easier if I just moved the last sentence to a regular log entry. How about:
```suggestion
		log "Factions" "Incipias" `The Incipias are a species of aliens that live to the northeast of Hai space. They originated on a gas giant, and resemble large, floating octopi. As they have not yet developed the hyperdrive, they are confined to a single system and are under the protection of the Quarg.`
```